helm/i40-aas/templates/initializer.yaml

```diff
@@ -9,6 +10,7 @@ metadata:
 spec:
   template:
     spec:
+      restartPolicy: OnFailure
```

Comment:
It's specific to a "Job" resource in Kubernetes. A Job is a container that is only supposed to run once and then exit successfully, instead of the usual eternal lifetime of a Deployment for instance. "restartPolicy" defines what happens when the container terminates. Here, it can be set to "OnFailure" meaning the container would restart if it terminated due to an error but not else or "Never" meaning it would only try to run once, irregardless of its success.

Reply:
Ok. i see. The container for the initializer is terminated after execution right? We should avoid a second execution of this after starting the AAS-Service

Reply:
Yes, this is exactly what this does. It would only run a second time, when it failed on the first time. Even this is probably up to debate, since it can cause the container or some parts of it to run in a crash loop. Maybe "never" would be better. But the service should be able to handle duplicate calls themselves.